tests/topotests/bfd_vrflite_topo1/r1/bfdd.conf

```diff
@@ -0,0 +1,16 @@
+!
+! debug bfd network
+! debug bfd peer
+! debug bfd zebra
+!
+bfd
+ peer 192.168.0.2 vrf vrf1
```

Comment:
The default BFD timers are too tight for CI under heavy load. I think we should use at least 1 second receive/transmit interval to avoid intermittent failures.

Reply:
ok